### PR TITLE
added type definitions

### PR DESCRIPTION
**Pull request by bigLucas**
I needed to create the type definitions for this library for a project with TypeScript and would like to share the definitions.

### CODE REVIEW

**Comment by ziadkh0** [comment]
Thank you for this PR :smile:

File: moment.easter.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import * as moment from 'moment';
+
+declare module 'moment' {
+    interface Moment {
+        easter(year?: number): Moment;
```
Comment (ziadkh0) [inline moment.easter.d.ts]: Are you sure there's a year argument?
It's been a very long time since I used this so I'm not sure

Comment (bigLucas) [reply moment.easter.d.ts]: Yes, there are two basic ways to use a function, according to README.
1. moment.easter(2014).format('YYYY/MM/DD');
2. moment({y: 2015}).easter().format('YYYY/MM/DD');

In the first case we need the year argument. In the second we take the year of the object already instantiated from the moment, with this.

Comment (ziadkh0) [reply moment.easter.d.ts]: Am I correct in assuming that the first case corresponds to the `export` below and the second to the `interface` above?

Comment (bigLucas) [reply moment.easter.d.ts]: Yes, you're right.

Comment (ziadkh0) [reply moment.easter.d.ts]: In that case, this method doesn't have a `year` argument

Comment (bigLucas) [reply moment.easter.d.ts]: OHH, sorry... I tried to leave it as an optional arg, but the implementation doesn't follow that.
**I'm push a new commit to fix this...**

**Pull request merged**